.github/workflows/tox.yaml

```diff
@@ -49,17 +49,21 @@ jobs:
       fail-fast: false  # Run all the tests to their conclusions.
       matrix:
         platform: [ubuntu-20.04, ubuntu-22.04]
+        python_version: ["3.8", "3.10"]
```

Comment:
Remind me to update which Github workflows are required when this is ready to merge.

Reply:
Removing:
- `tests (ubuntu-20.04)`
- `tests (ubuntu-22.04)`

Adding:
- `tests (ubuntu-20.04, 3.8)`
- `tests (ubuntu-20.04, 3.10)`
- `tests (ubuntu-22.04, 3.8)`
- `tests (ubuntu-22.04, 3.10)`